pootle/templates/browser/index.html

```diff
@@ -143,10 +143,16 @@ <h3 class="top">{% trans "Updates" %}</h3>
 {% endif %}
 
 <div id="autorefresh-notice">
-  {% blocktrans trimmed %}
-  Some data on this page is currently being calculated, and the page will be
-  refreshed automatically in <strong>30 seconds</strong>.
-  {% endblocktrans %}
+  {% if stats_banner_auto_refresh_enabled %}
+    {% blocktrans trimmed %}
+    Some data on this page is currently being calculated, and the page will be
+    refreshed automatically in <strong>30 seconds</strong>.
+    {% endblocktrans %}
+  {% elif has_site_admin_permission %}
+    {% trans "Stats are out of date. See documentation for dealing with stats issues." %}
+  {% else %}
+    {% trans "The stats on this page may be out of date." %}
+  {% endif %}
```

Comment:
I think it is better to check first if user is superuser, and if not then check if the setting is enabled or not (I don't think Evernote disagrees on showing a different message to site admins):

```
  {% if user.is_superuser %}
    {% trans "Stats are out of date. See documentation for dealing with stats issues." %}
  {% elif stats_banner_auto_refresh_enabled %}
    {% blocktrans trimmed %}
    Some data on this page is currently being calculated, and the page will be
    refreshed automatically in <strong>30 seconds</strong>.
    {% endblocktrans %}
  {% else %}
    {% trans "The stats on this page may be out of date." %}
  {% endif %}
```